src/main/java/org/babi/backend/place/api/PlaceController.java

```diff
@@ -0,0 +1,68 @@
+package org.babi.backend.place.api;
+
+import org.babi.backend.common.dao.PageableResponse;
+import org.babi.backend.place.dao.PlaceCriteria;
+import org.babi.backend.place.domain.Place;
+import org.babi.backend.place.service.PlaceService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/places")
+public class PlaceController {
+
+    private final PlaceService placeService;
+    private final List<SavePlaceApiPreProcessor> savePlaceApiPreProcessors;
+
+    @Autowired
```

Comment:
you can remove @Autowired for constructor

Reply:
Yeah, I can, but it's only to let people know who don't get about one single constructor injection how does dependency injection play.